tsconfig.json

```diff
@@ -7,7 +7,12 @@
     "strict": true,
     "target": "es2022",
     "moduleResolution": "Node",
-    "composite": true
+    "composite": true,
+    "resolveJsonModule": true,
+    "noEmit": true,
```

Comment:
Mm if we don't emit the transpiled JS files, Oclif will not work as is expecting some files to be in the `dist` directory, does this prevents that directory to be filled with the build files?